src/main/java/com/unityTest/courseManagement/models/AssignmentAttributeName.java

```diff
@@ -0,0 +1,10 @@
+package com.unityTest.courseManagement.models;
+
+/**
+ * Attributes of an assignment Names should match those returned as part of AssignmentAttributeView
```

Comment:
I'm missing some punctuation on my description you copied here, if you don't mind adding it 🙏